components/dialog/dialog-mixin.js

```diff
@@ -105,16 +96,16 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 		super.updated(changedProperties);
 		if (!changedProperties.has('opened')) return;
 
-		const dialogService = await getIfrauDialogService();
+		ifrauDialogService = await getIfrauBackdropService(ifrauDialogService);
```

Comment:
Related to the comment down in the helper method, I think you can remove the local `let ifrauDialogService` at the top of this file and change this to:

```javascript
const ifrauBackdropService = tryGetIfrauBackdropService();
```

Reply:
The reason I wanted to pass in and keep the ifrauDialogService is so that it only needs to be fetched once. The getIfrauBackdropService function will return the ifrauDialogService if it's already defined. I'm not sure if there would be a performance implication with fetching the service every time the dialog is opened. 🤔

Reply:
Understood, but the helper itself can do that by having the local variable in its file. That way it can dedupe calls across dialogs and dropdowns (and anything else we add this to in the future).

Reply:
Aha, makes sense! Will do.